tests/integration/import_csv_file.t

```diff
@@ -42,20 +42,40 @@ sub fake_download_image ($) {
 	return $response;
 }
 
+my @tests = (
+	{
+		test_case => "test",
+		csv_file => "test.csv",
+	},
+	{
+		test_case => "replace_existing_values_1",
```

Comment:
For this test case, you want to import 2 CSV files, so instead of having a csv_file argument, you could have a csv_files argument:

```
my @tests = (
	{
		test_case => "test",
		csv_files => ["test.csv"],
	},
	{
		test_case => "replace_existing_values",
        csv_files => ["replace_existing_values_1.csv", "replace_existing_values_1.csv"],
},
```

You cannot have the replace_existing_values 1 and 2 in 2 different tests cases, because when we iterate on the test cases, one of the first things we do is remove_all_products()